client/my-sites/plans-features-main/index.tsx

```diff
@@ -263,6 +273,7 @@ const PlansFeaturesMain = ( {
 	isSpotlightOnCurrentPlan,
 }: PlansFeaturesMainProps ) => {
 	const [ isFreePlanPaidDomainDialogOpen, setIsFreePlanPaidDomainDialogOpen ] = useState( false );
+	const [ isFreeFreeUpsellOpen, setIsFreeFreeUpsellOpen ] = useState( false );
```

Comment:
Non-blocking:

Did we want to rename the state here to freePlanFreeDomain as well?

Reply:
Yes I am hoping to address this once I ship the experiment :)